python/ray/tests/kuberay/scripts/scale_up_custom.py

```diff
@@ -1,15 +1,29 @@
+import os
+
+import pytest
+
 import ray
 
 
 def main():
-    """Submits custom resource request."""
+    """Submits custom resource request.
+
+    Also, validates runtime env data submitted with the Ray Job that executes
+    this script.
+    """
+    assert pytest.__version__ == "6.0.0"
```

Comment:
This is used to make sure it matches with job submission dependencies? or it is the code dependency?

Reply:
> make sure it matches with job submission dependencies

Yep. Will add a comment.

Reply:
The test logic is maybe a bit unfocused.